my-app/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react'
+import logo from './logo.svg'
+import './App.css';
+
+class Board extends Component {
+  constructor() {
+    super();
+    this.size = 9;
+    this.state = {
+
+      // wIsNext: true,
+      board: this.createGameBoard(this.size),
+
+    };
+  }
+
+  createGameBoard(size){
+    let gameBoard = [];
+      for(let x = 0; x < size; x++){
+        gameBoard[x] = []
+        for (let y = 0; y < size; y++){
+          gameBoard[x][y] = ['Y:',y,',','X:',x]
+        }
+    }
+
+    return gameBoard
+  }
+
+  handleClick(x,y) {
+    const squares = this.state.squares.slice();
```

Comment:
I'm not sure I understand the purpose of `slice` here... Having said that, I would be using an instance of `GameBoard`, and use functions on the instance to update the board state - this component knows too much about what boards are, and is taking on too much responsibility in both managing the state of the board, and rendering that state.